django_project/lesson/admin.py

```diff
@@ -0,0 +1,40 @@
+# coding=UTF-8
+"""Model admin class definitions."""
+
+from django.contrib import admin
+from lesson.models.answer import Answer
+from lesson.models.further_reading import FurtherReading
+from lesson.models.section import Section
+from lesson.models.specification import Specification
+from lesson.models.worksheet import Worksheet
+from lesson.models.worksheet_question import WorksheetQuestion
+
+
+class AnswerAdmin(admin.ModelAdmin):
```

Comment:
shouldn't these get at least a 

pass

in their bodies?

Reply:
Added.